src/pages/ALLO_ManageAllocations.page

```diff
@@ -45,9 +45,12 @@
                         thisRowPercent.prop('disabled', true);
                     }
                 }
-                if (!isNaN($(o).val()))
-                    sumAmount += Math.round($(o).val()*100);
-               
+                var amt=$(o).val();
+                // due to how chrome handle's the global replace, have to
+                // first remove the currency symbols first and then strip out all commas.
+                if (!isNaN(amt.replace(/[\$£€]/g, '').replace(/,/g, '')))
+                    sumAmount += Math.round(amt.replace(/[\$£€]/g, '').replace(/,/g, '')*100);
```

Comment:
Good point. Completely forgot that a comma is used for that in some countries. I think there's a way I can determine the appropriate thousands delimiter (probably in Apex). If so, we can use that the JS to strip out the appropriate character to avoid removing a comma when it's actually meant to be a decimal delimiter.